src/linux.rs

```diff
@@ -0,0 +1,912 @@
+use crate::{ CtlValue, SysctlError, };
+
+use libc::{ c_int, c_void, c_ulong, size_t, };
+
+use std::fs::OpenOptions;
+use std::io::{ self, Write, Read, };
+
+
+/// Takes the name of the OID as argument and returns
+/// a result containing the sysctl value if success,
+/// or a SysctlError on failure
+///
+/// # Example
+/// ```
+/// extern crate sysctl;
+///
+/// fn main() {
+///     println!("Value: {:?}", sysctl::value("kern.osrevision"));
```

Comment:
This entry does not exist on Linux. Please update to match something available on Linux.

Reply:
done.